package/Gemfile

```diff
@@ -7,6 +7,7 @@ gem 'json'
 gem 'fluentd'
 
 # Input / Output plugins:
+gem 'fluent-plugin-concat'
```

Comment:
could you add a comment for filter plugins: and place gem 'fluent-plugin-concat' there

Reply:
Looking forward to this! 

Down the line it wold be fantastic to specify the multiline_start_regexp parameter in the rancher logging GUI.